openslides/core/static/js/core/base.js

```diff
@@ -553,27 +553,57 @@ angular.module('OpenSlidesApp.core', [
 ])
 
 // Template hooks
-// 2 possible uses:
-// - { Id: 'myHookId', template: '<button>click me</button>' }
-// - { Id: 'myHookId', templateUrl: '/static/templates/plugin_name/my-hook.html' }
-// It is possible to provide a scope, that is merged into the scope of the templateHook.
-// This overrides functions/values of the parent scope, but there may are conflicts
-// with other plugins defining the same function/value. E.g.:
+// Possible uses:
+// 1. { id: 'myHookId', template: '<button>click me</button>' }
+// 2. { id: 'myHookId', templateUrl: '/static/templates/plugin_name/my-hook.html' }
+// 3. { id: 'myHookId' }
+//
+// Deprecated: Give the id with 'Id'. Please use 'id'.
+//
+// Option 3 is for just changing the scope (see below), but not the original content. This
+// is usefull to alter a JS behavior, e.g. on a ng-click. In this case, override is false
+// for this template hook.
+//
+// It is possible to provide a scope, that is merged into the surrounding scope.
+// You can override functions or values of the surrounding scope by providing them:
 // { Id: 'hookId', template: '<button ng-click="customFn()">click me</button>',
```

Comment:
Use `id` instead of `Id`. Also use `customOrOverwritten` instead of `customFn`, right?